aries_fzn/src/cli.rs

```diff
@@ -28,10 +70,17 @@ pub struct Args {
     #[arg(short, long)]
     pub intermediate: bool,
 
-    /// Solver configuration.
-    #[arg(short, long, value_enum)]
-    #[arg(default_value_t)]
-    pub config: Config,
+    /// Variable order.
+    #[arg(long, default_value = "lexical", value_name = "ORDER", value_parser = parse_var_order)]
```

Comment:
Isn't this a very suboptimal choice ?
The default should always be the best / most reliable choice